src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/LanguageServices/GeneratedCodeRecognition/AbstractGeneratedCodeRecognitionService.cs

```diff
@@ -28,18 +28,8 @@ public async Task<bool> IsGeneratedCodeAsync(Document document, CancellationToke
 
         private static bool IsGeneratedCode(SyntaxTree syntaxTree, Document document, CancellationToken cancellationToken)
```

Comment:
❔ Can we inline this to avoid duplication?

Reply:
I am not sure I understood, what part of duplication is avoided by inlining? This is invoked at couple of places above.